packages/elements/src/button/button.spec.js

```diff
@@ -0,0 +1,27 @@
+import { mount } from 'enzyme'
+import { Button } from './button.component.js'
+
+describe('<Button>', () => {
```

Comment:
It would be great if we could have visual regression tests for elements, since their main purpose is style. Otherwise it's hard to do anything but test implementation details, which is more labour intensive and a little more fragile.

Maybe until we have those we should just test the hard requirements, like passing a ref, spreading props, and omit testing styles.

Reply:
@ismay :100: since these components exist for styling, visual regression would be the most efficient way of testing them

Reply:
https://www.chromatic.com/pricing it does say "free for open source" here. It's a testing service (that includes visual tests) created by the storybook team. I do think that that'd be a good fit for ui.